mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/query/SortConverter.java

```diff
@@ -55,7 +55,7 @@ private static FieldSortBuilder getSortClause(SearchQuery.Sort.SortClause clause
             case Size :
                 return SortBuilders.fieldSort(JsonMessageConstants.SIZE);
             case SentDate :
-                return SortBuilders.fieldSort(JsonMessageConstants.SENT_DATE);
+                return SortBuilders.fieldSort(JsonMessageConstants.DATE);
```

Comment:
Well... Won't it break sorting on Sent date, when there is a **Date** header? 

In that case, don't we want to sort on SentDate?

If it pass integration tests my bet is that you are being lucky...